src/platforms/shared/styles/common/hidden-ad-helper.scss

```
@@ -0,0 +1,6 @@
+@mixin hidden-ad-helper() {
```

```suggestion
@mixin hidden-ad() {
```

or you can add this to _mixin.scss

---

moved :)